packages/ocp-index-top/ocp-index-top.0.4.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+name: "ocp-index-top"
+version: "0.4.1"
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+author: "Reynir Björnsson <reynir@reynir.dk>"
+dev-repo: "https://github.com/reynir/ocp-index-top.git"
+homepage: "https://github.com/reynir/ocp-index-top/"
+bug-reports:  "https://github.com/reynir/ocp-index-top/issues"
+doc: "https://reynir.github.io/ocp-index-top/"
+license: "BSD-2-clause"
+build: [
+  "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%"
+]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
+  "cppo" {build}
+  "ocp-index"
+]
```

Comment:
Thanks very much for this package! It is almost ready for merge... it fails to build on OCaml 4.01.0, so you need a constraint like:

```
available: [ ocaml-version > "4.01.0" ]
```

this is the failure:
```
# File "ocpIndexTop.cppo.ml", line 71, characters 11-20:
# Error: Syntax error: operator expected.
# Command exited with code 2.
```